tests/travis/create-mysql-schema.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+set -ex
+
+mysql -e "CREATE SCHEMA doctrine_tests; GRANT ALL PRIVILEGES ON doctrine_tests.* to travis@'%'"
+mysql -e "CREATE SCHEMA test_create_database; GRANT ALL PRIVILEGES ON test_create_database.* to travis@'%'";
+mysql -e "CREATE SCHEMA test_drop_database; GRANT ALL PRIVILEGES ON test_drop_database.* to travis@'%'";
```

Comment:
This is mostly sql. How about creating an sql file containing these 3 queries and executing in just one `mysql` call?